packages/server-boilerplate/src/utils/initialiseApiClient.ts

```diff
@@ -66,7 +66,7 @@ const upsertApiClient = async ({
   password: string;
   salt: string;
 }) => {
-  const secretKeyHash = encryptPassword(password, salt);
+  const secretKeyHash = await encryptPassword(password, salt);
```

Comment:
When the api clients start up, the api client hashes in the database are re-created so we don't need to manually migrate them.